articles/reactive-skipwhile.md

```diff
@@ -4,3 +4,9 @@ title: SkipWhile
 ---
 
 ![Marble diagram](~/images/reactive-skipwhile.svg)
+
```

Comment:
Just a side note. We should revisit the idea of a RepeatWhile.

Reply:
There is already a PR for that: https://github.com/bonsai-rx/bonsai/pull/1069

We should add it to the agenda to discuss whether to include for 2.8.